pyfdb/pyfdb.py

```diff
@@ -168,15 +168,15 @@ class ListIterator:
     __iterator = None
     __key = False
 
-    def __init__(self, fdb, request, duplicates, key=False, expand=True):
```

Comment:
Is there a particular reason to set the argument to 3 in case of not providing it? I'd assume that 3 currently means everything? Is there a way to return everything via the fdb api?

Reply:
I know there is documentation for the function missing, but could you add it and also add it for the new parameter?